grumblr/config/application.rb

```diff
@@ -6,7 +6,7 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
-module Grumblr
+module Grumble
```

Comment:
I recommend that you not change this.  AFAIK, this module name is used in multiple locations.  The app is Grumblr and it manages Grumbles.